### PR TITLE
[Homework #19] - react-api by Brisa Rojas

### PR DESCRIPTION
**Pull request by bytheseaside**
Hi Ilya, here's my homework 

I just have something left to do, regarding the % button on calculator, what's supposed to do? Calculate percentages or module operation like '3%2 =1 ' in JS? If it's percentages, how is it exactly? Let's say I have a%b, should I calculate b% of a or the other way around

### CODE REVIEW

**Comment by bytheseaside** [comment]
I added the remainder of division functionality for the percentage button and onKeyDown event to make it possible to also use it with the keyboard

File: homeworks/brisa-rojas/19-react-api/src/components/CalcButton/CalcButton.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { alpha, styled } from '@mui/material/styles';
+import Button from '@mui/material/Button';
+import { red } from '@mui/material/colors';
```
Comment (ilya-ischenko) [inline homeworks/brisa-rojas/19-react-api/src/components/CalcButton/CalcButton.js:3]: I see unnecessary imports there

Comment (bytheseaside) [reply homeworks/brisa-rojas/19-react-api/src/components/CalcButton/CalcButton.js]: I'm sorry, I used it previously in some styles but then changed them and didn't delete the imports. I already corrected it

**Review by ilya-ischenko — APPROVED** [review]
Well done!
Approved.

**Pull request merged**